package.json

```diff
@@ -21,6 +21,7 @@
     "dot-object": "^2.1.3",
     "express": "^4.17.1",
     "firebase": "^7.14.0",
+    "firebase-functions": "^3.6.1",
```

Comment:
Do we need this?

Reply:
yes, for the running of emulator

Reply:
oops i also added it in another pr per request from Tuan thats gonna be a potential clash hopefully this PR can land first though